configuration.k

```diff
@@ -25,6 +25,11 @@ imports SOLIDITY-SYNTAX
 
 
                 <vposition> 1:Int </vposition>
+                <loopCon> true </loopCon>
+                <funCond> true </funCond>
```

Comment:
Use `Con` or `Cond` for "condition"? Shall we make it the same? (similar comment applies in other places too).